### ISSUE TITLE
Long term uploading

### ISSUE THREAD

**Issue by Elisagig** [Mergpijp/GlobalOrthodoxy#128]
Some colleagues are complaining that: The uploading of the images takes long... if you go to Edit and want to see the files:

**Comment by Woseseltops** [comment]
Hey @Elisagig , I guess you mean 'loading' instead of 'uploading', right? That is, if you investigate 1 individual publication, it takes some time before the images appears? 

If so, I guess we could make automatically smaller versions of all images so they show faster in the list.

**Comment by Elisagig** [comment]
Yes true. So loading is the term for 1 publication? Good to know :) 

Does smaller versions make the image quality worse?

**Comment by Mergpijp** [comment]
Yes image quality get worse but not to bad these times when you use online image size shortener. This might be fixed with #136?
